examples/photoframe/source/src/org/kaaproject/kaa/demo/photoframe/PhotoFrameController.java

```diff
@@ -0,0 +1,360 @@
+/*
+ * Copyright 2014-2015 CyberVision, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kaaproject.kaa.demo.photoframe;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.kaaproject.kaa.client.KaaClient;
+import org.kaaproject.kaa.client.event.EndpointKeyHash;
+import org.kaaproject.kaa.client.event.registration.OnDetachEndpointOperationCallback;
+import org.kaaproject.kaa.client.event.registration.UserAttachCallback;
+import org.kaaproject.kaa.common.endpoint.gen.SyncResponseResultType;
+import org.kaaproject.kaa.common.endpoint.gen.UserAttachResponse;
+import org.kaaproject.kaa.demo.photoframe.PhotoFrameEventClassFamily.DefaultEventFamilyListener;
+import org.kaaproject.kaa.demo.photoframe.event.AlbumListEvent;
+import org.kaaproject.kaa.demo.photoframe.event.DeviceInfoEvent;
+import org.kaaproject.kaa.demo.photoframe.event.PlayAlbumEvent;
+import org.kaaproject.kaa.demo.photoframe.event.PlayInfoEvent;
+import org.kaaproject.kaa.demo.photoframe.event.StopPlayEvent;
+import org.kaaproject.kaa.demo.photoframe.event.UserAttachEvent;
+import org.kaaproject.kaa.demo.photoframe.event.UserDetachEvent;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.net.Uri;
+import android.provider.MediaStore;
+import de.greenrobot.event.EventBus;
+
```

Comment:
This class need some basic description/comments. Smth like:
This class receives Kaa events and user attach/detach callbacks and dispatch them to other application components via event bus. It is also responsible for fetching albums from android {@link MediaStore}.